app/controllers/admin/stats/contributions_controller.rb

```diff
@@ -1,5 +1,11 @@
 class Admin::Stats::ContributionsController < AdminController
   def index
     @result = Rails.cache.fetch(Cache::EyeshadeStatsJob::EYESHADE_CONTRIBUTION_TOTALS) || []
+    @publishers = Publisher.where(id: @result.map { |entry| entry['account_id'].split(":")[1] }).load
```

Comment:
i'm a sucker for user Rails-esque style
`entry['account_id'].split(":").second`

Reply:
eh, on second thought nvm. the `[1]` is pretty prevalent throughout this codebase